trunk/SUAVE/Input_Output/OpenVSP/write_vsp_propeller.py

```diff
@@ -0,0 +1,197 @@
+## @ingroup Input_Output-OpenVSP
+# write_vsp_propeller.py
+# 
+# Created: Feb 2021, M. Clarke
+# ----------------------------------------------------------------------
+#  Imports
+# ---------------------------------------------------------------------- 
+from SUAVE.Core import Units, Data
+from SUAVE.Methods.Aerodynamics.AVL.purge_files import purge_files 
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.import_airfoil_geometry\
+     import import_airfoil_geometry
+import numpy as np
+import shutil 
```

Comment:
Not sure what this import is but it isn't used anywhere in this file

Reply:
removed